cli/internal/run/dry_run.go

```diff
@@ -102,10 +102,11 @@ func executeDryRun(ctx gocontext.Context, engine *core.Engine, g *graph.Complete
 			return err
 		}
 
-		command, ok := packageTask.Command()
-		if !ok {
-			command = "<NONEXISTENT>"
+		command := "<NONEXISTENT>"
```

Comment:
Is it worth extracting this var out while we're here? I think a variable name would help me understand what this value signifies.

Reply:
for sure! pushed up a commit to explain. Doing to set automerge on this PR, but if it stil doesn't make sense, let me know and we can figure out how to improve the commends / naming.